cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraRDDWriter.scala

```diff
@@ -148,8 +148,7 @@ object CassandraRDDWriter {
                 })
               }
 
-              val results = (rows flatMap elaborateRow flatMap rowToBytes map retire)
-                .join(threads)
+              val results = (rows flatMap elaborateRow flatMap rowToBytes flatMap retire)
```

Comment:
would this be a `join(threads)` equivalent? i was not sure, that's why decided to make it clear, to keep things simple.

Reply:
Or probably it's a `1.0` feature for sure.

Reply:
Semantically equivalent, I believe so, but we lose some parallelism here

Reply:
We loose some parallelism? What do you mean? You mean that the behaviour would differ significantly?

Reply:
@moradology don't you want to use explicit `.parJoin(threads)` instead of a flatMap on the last step?

Reply:
```scala
implicit val cs: ContextShift[IO] = IO.contextShift(ec)
(index map readRecord)
        .parJoin(threads)
        .compile
        .toVector
        .unsafeRunSync.flatten
```

Reply:
@moradology no, it is a question for you, as you introduced this change, i was just asking :D